portfolio/src/main/java/com/google/sps/servlets/LogInServlet.java

```
@@ -0,0 +1,40 @@
+/* Servlet that manages log in behavior of webpage */
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.data.Login;
+
+@WebServlet("/login")
+public class LogInServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    UserService userService = UserServiceFactory.getUserService();
+  
+    
+    String emailAddress = null;
+    try {
+      emailAddress = userService.getCurrentUser().getEmail();
+    } catch(NullPointerException e) { }
```

Empty catch blocks are considered a bad practice. Let's instead check if getCurrentUser() returns null and call getEmail() only if it does not. https://www.tutorialspoint.com/can-we-have-an-empty-catch-block-in-java#:~:text=Yes%2C%20we%20can%20have%20an,caught%20by%20the%20catch%20block.